README.md

```diff
@@ -140,6 +140,45 @@ For a fair comparison, [LogLevel](http://npm.im/loglevel) was extended
 to include a timestamp and [bole](http://npm.im/bole) had
 `fastTime` mode switched on.
 
+<a name="transports"></a>
+## Transports
+
+A transport in most logging libraries is something that runs in-process to
+perform some operation with the finalized log line. For example, a transport
+might send the log line to a standard syslog server after processing the log
+line and reformatting it. For details on implementing, and some already written,
+transports, see our [Transports⇗](docs/transports.md) document.
+
+> **Pino *does not* natively support in-process transports.**
+
+Pino does not support in-process transports because Node processes are
+single threaded processes (ignoring some technical details). In order for Pino
+to achieve the speed it achieves, Pino purposefully offloads the handling of
+logs, and their ultimate destination, to external processes so that the
+threading capabilities of the OS can be used (or other CPUs).
+
+One consequence of this methodology is that "error" logs do not get written to
+`stderr`, as might be exepected. However, since Pino logs are in a parseable
+format, it is possible to use tools like [pino-tee][pino-tee] or [jq][jq]
+to work with the logs. For example, to view only logs marked as "error" logs:
+
+```
+$ node an-app.js | jq 'select(.level == 50)'
+```
+
+In short, the way Pino generates logs:
+
+1. Reduces the impact of logging on your application to an extremely minimal amount.
+2. Gives greater flexibility in how logs are processed and stored.
+
+Given all of the above, Pino clearly promotes out-of-process log processing.
```

Comment:
I'm trying to directly respond to questions around "transports" that have been raised multiple times. So I'm trying to give strong reasons for the way things are. Does that make sense?